### PR TITLE
Process partition resize update before return

### PR DESCRIPTION
**Pull request by pothos**
During Partition.Resize() the device node shortly
disappears and bubbles up again through udev and
the interfaces of the object are set up anew.

The method will now wait for the partition
property to be updated with the new size.

This depends on the unreleased
https://github.com/storaged-project/libblockdev/commit/7ffc9746f77d96d06c497e260d3ea042c52e1888

### CODE REVIEW

**Comment by vojtechtrefny** [comment]
Jenkins, ok to test.

File: src/udiskslinuxpartition.c [hunk 1]
```diff
@@ -812,13 +851,40 @@ handle_resize (UDisksPartition       *partition,
       goto out;
     }
 
+  /* Wait for partition property to be updated so that the partition interface
+   * will not disappear shortly after this method returns.
+   * Clients could either wait explicitly wait for an interface or try
```
Comment (vpodzime) [inline src/udiskslinuxpartition.c]: double *wait*

**Comment by pothos** [comment]
Is there anything left to do to get it into the release?

**Comment by vpodzime** [comment]
> Is there anything left to do to get it into the release?

Nope, thanks!

**Pull request merged**